mantidproject/mantid
# Remove Run.h and Sample.h #include from ExperimentInfo.h

This is a small improvement of the number of automatic includes most of our algorithms get.

Before:

```
Number of cpp files                          :   1627
Total recursive number of includes in files  : 189246
Average number of includes in cpp files      :    116
```

After:

```
Number of cpp files                          :   1627
Total recursive number of includes in files  : 181689
Average number of includes in cpp files      :    111
```

I did not see a noticeable improvement of compile times, but probably this change is simply too small.

**To test:**

Should be fine if build passes, maybe a very quick code review to ensure that I did not accidentally delete anything.

This is a small part for #15319.

No release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
